src/vs/workbench/services/untitled/common/untitledTextEditorInput.ts

```diff
@@ -20,6 +20,7 @@ import { IFilesConfigurationService } from 'vs/workbench/services/filesConfigura
 import { ITextModelService } from 'vs/editor/common/services/resolverService';
 import { DisposableStore, dispose, IReference } from 'vs/base/common/lifecycle';
 import { ITextResourceConfigurationService } from 'vs/editor/common/services/textResourceConfiguration';
+import { ICustomEditorLabelService } from 'vs/workbench/common/editor/editorLabels';
```

Comment:
Since this is now a service, I would have it at `vs/workbench/services/editor/common/customEditorLabelService`